source/compiler-core/slang-diagnostic-sink.h

```diff
@@ -138,8 +138,9 @@ class DiagnosticSink
     {
         enum Enum: Flags
         {
-            VerbosePath = 0x1,                  ///< Will display a more verbose path (if available) - such as a canonical or absolute path
-            SourceLocationLine = 0x2,           ///< If set will display the location line if source is available
+            VerbosePath         = 0x1,           ///< Will display a more verbose path (if available) - such as a canonical or absolute path
+            SourceLocationLine  = 0x2,           ///< If set will display the location line if source is available
+            HumaneLoc           = 0x4,           ///< If set will display humane locs (filename/line number) information
```

Comment:
It seems like the `HumaneLoc` part doesn't specifically deal with humane locations, but rather with whether source locations are output *at all*.

(We probably need to go through and clarify all our terminology around source locations so that the compiler is always consistent about what things like "humane" locations even mean, but that's clearly out of scope here)